src/test/groovy/com/palantir/gradle/javadist/JavaDistributionPluginTests.groovy

```diff
@@ -62,10 +62,13 @@ class JavaDistributionPluginTests extends Specification {
 
         // try all of the service commands
         exec('dist/service-name-0.1/service/bin/init.sh', 'start') ==~ /(?m)Running 'service-name'\.\.\.\s+Started \(\d+\)\n/
+        sleep 5000
```

Comment:
Have you actually found this to be flakey? The sleeps shouldn't be necessary as the shell execution waits for the program to actually run, and the program is just a long sleep anyway.